packages/common/store/index.js

```diff
@@ -21,5 +24,10 @@ if (module.hot) {
   });
 }
 
-export * from './selectors';
+export {
+  storage,
+  store,
+  persistedStore,
+};
+
 export default store;
```

Comment:
@soroushm Since store is exported as a named component above, why is it still exported as a default here, when it's not imported by default anywhere else